app/controllers/reactions_controller.rb

```diff
@@ -103,19 +103,31 @@ def cached_user_positive_reactions(user)
 
   private
 
-  def reaction_user(reaction)
-    if reaction.reactable_type == "User"
-      reaction.reactable
-    else
-      reaction.reactable.user
-    end
+  def create_reaction(category)
```

Comment:
Can I suggest `build_reaction` as this doesn't actually save it to the DB?

Reply:
Good call! Done!